modules/tensor_mechanics/test/tests/torque_reaction/disp_about_axis_axial_motion.i

```diff
@@ -176,24 +100,17 @@
   l_max_its = 30
   nl_max_its = 20
   nl_rel_tol = 1e-10
-  nl_abs_tol = 1e-12
+  nl_abs_tol = 1e-9
   l_tol = 1e-8
 
   start_time = 0.0
-  dt = 0.1
-  dtmin = 0.1 # die instead of cutting the timestep
+  dt = 1
+  dtmin = 1 # die instead of cutting the timestep
 
-  end_time = 0.5
+  end_time = 90
```

Comment:
This test still remains under the two second mark with this large of an end_time?

Reply:
Yes, it runs fast. I'll verify, but I'm pretty sure its well under a second.

Reply:
It took 1.5 s.  I doubled the time step size, and now it runs in .677s.